student/tests.py

```diff
@@ -1,15 +1,84 @@
 import pytest
+from django.contrib.auth.models import User
+from django.db.models import QuerySet
 
+from student.models import Student, EducationalInstitution
+from datetime import date
 
-def test_basic_math_operation():
-    assert 10 == 2 * 5
+
+@pytest.fixture
+def example_institution_a():
+    institution = EducationalInstitution(name="A")
+    institution.save()
+    return institution
+
+
+@pytest.fixture
+def example_institution_b():
+    institution = EducationalInstitution(name="B")
+    institution.save()
+    return institution
+
+
+@pytest.fixture
+def example_student_a(example_institution_a):
+    example_user = User.objects.create_user("test", "testpassword")
+    example_user.save()
+    student = Student(user=example_user, full_name="test user", email="test@test.com", date_of_birth=date(2020, 7, 13),
+                      phone_number="0000000000", educational_institution=example_institution_a,
+                      graduation_date=date(2020, 7, 13))
+    student.save()
+    return student
 
 
 @pytest.fixture
-def numbers_list():
-    return [1, 2, 3, 4]
+def example_student_b(example_institution_b):
+    example_user = User.objects.create_user("test2", "testpassword")
+    example_user.save()
+    student = Student(user=example_user, full_name="test user2", email="test2@test.com",
+                      date_of_birth=date(2023, 7, 13),
+                      phone_number="0000000000", educational_institution=example_institution_b,
+                      graduation_date=date(2023, 7, 13))
+    student.save()
+    return student
+
+
+@pytest.mark.django_db
+def test_get_institutions_with_enrolled_students_returns_institutions_with_enrolled_students_as_query_set(
```

Comment:
The name of the test is "test.. returns students **as a query set**", so in that case only first assertion on line 50 asserts this, but you also test here the student names and the institutions.
So either remove those assertions or rename the test.

Reply:
The name also says it returns institutions with enrolled students, and it checks that as well